MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+<td>Format</td>
+<td>Gets or sets the time format for Time Picker.</td>
+</tr>
+</table>
+
+### DataFormListItem
+
+The common properties of `Xamarin SfDataForm` list items (` DataFormPickerItem`, ` DataFormAutoCompleteItem`, ` DataFormDropDownItem` and ` DataFormRadioGroupItem` ) are added to [DataFormListItem] class in `.NET MAUI SfDataForm` 
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ItemsSource](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_ItemsSource)'| markdownify }}</td>
+<td>ItemsSource</td>
+<td>Gets or sets the collection which will get displayed in picker where the user can select from available items.</td>
+</tr>
+
+<tr>
+<td>{{'[DisplayMemberPath](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_DisplayMemberPath)'| markdownify }}</td>
+<td>DisplayMemberPath</td>
+<td>Gets or sets a path to a value on the source object that will get displayed in picker.</td>
+</tr>
+
+<tr>
+<td>{{'[ValueMemberPath](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_ValueMemberPath)'| markdownify }}</td>
+<td>SelectedValuePath</td>
+<td>Gets or sets a path to a value on the source object which will get saved to data field.</td>
+</tr>
+</table>
+
+### DataFormComboBoxItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Nil</td>
+<td>IsEditable</td>
+<td>Gets or sets a value indicating whether the ComboBox can be edited.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the text search mode for ComboBox editor.</td>
+</tr>
+</table>
+
+### DataFormAutoCompleteItem
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[AutoCompleteMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormAutoCompleteItem.html#Syncfusion_XForms_DataForm_DataFormAutoCompleteItem_AutoCompleteMode)'| markdownify }}</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the text search mode for auto complete editor.</td>
+</tr>
+</table>
+
+### DataFormDisplayOptionsAttribute
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ColumnSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ColumnSpan)'| markdownify }}</td>
+<td>ColumnSpan</td>
+<td>Gets or sets the number of column that this item spans to increase item's width.</td>
```

Comment:
<td>Gets or sets the number of columns this item spans to increase the item's width.</td>